config/locales/en.yml

```diff
@@ -132,7 +122,7 @@ en:
     either_way: Either way
     indictable: Indictable
     already_in_crown_court: Already in Crown Court
-    commital: Commital
+    committal: Committal
     appeal_to_crown_court: Appeal to Crown Court
     appeal_to_crown_court_with_changes: Appeal to Crown Court with charges
```

Comment:
Unrelated to the changes in this PR but I've noticed on Apply we have slightly different copy for some of these case types. Maybe worth checking if it also applies to Review?
Here are the differences:

```
already_in_crown_court: Trial already in crown court
committal: Committal for sentence
appeal_to_crown_court: Appeal to crown court
appeal_to_crown_court_with_changes: Appeal to crown court with changes in financial circumstances
```

Reply:
Very helpful, thank you. I'll double check the designs